### ISSUE TITLE
File '<app>/node_modules/botframework-webchat/src/types/JSpeech.d.ts' not found.

### ISSUE THREAD

**Issue by rajshrimohanks** [Microsoft/BotFramework-WebChat#944]
I'm trying to use this package with my Angular 5 app. I installed it as:
```
npm install botframework-webchat --save
```

I had some errors pop up when I tried to run complaining about type definitions missing for Reach. So I did a `npm install --save @types/react` which fixed them.

However, the following error still persists:
```
ERROR in node_modules/botframework-webchat/built/SpeechModule.d.ts(1,22): error TS6053: File '/home/rajshrimohanks/Documents/dev/myapp/node_modules/botframework-webchat/src/types/JSpeech.d.ts' not found.
```
And sure enough, the specified file (the whole of the `src/` folder in fact) does not exist in `node_modules`. I believe that this is a bug?

**Comment by christian-vorhemus** [comment]
I just came across the same problem. As a workaround you can try to install version 0.12.0 until this is fixed.

**Comment by compulim** [comment]
Sorry folks, this is fixed in #966 yesterday.
